### PR TITLE
fwlite 11.0.0 retry with boost 1.70.0

### PR DESCRIPTION
**Pull request by gartung**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by gartung** [comment]
@chrisburr I needed genreflex so I changed the dependency to root but I am getting many conflicts. When I add root-dependencies as a dependency the conflicts are more limited:

```
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                              
failed to parse packages from exception: The following specifications were found to be incompatible with each other:                
                                                                                                                                   
Package icu conflicts for:
root -> libxml2[version='>=2.9.8,<2.10.0a0'] -> icu[version='56.*|58.*|>=58.2,<59.0a0|>=64.2,<65.0a0']
root-dependencies -> libxml2[version='>=2.9.10,<2.10.0a0'] -> icu[version='>=64.2,<65.0a0']
xerces-c -> icu[version='56.*|58.*|>=58.2,<59.0a0|>=64.2,<65.0a0']
libboost -> icu[version='>=58.2,<59.0a0']
```

**Comment by gartung** [comment]
Even after removing xerces-c I still get a conflict over the icu version
CMS is using eos portable archive in ConditionFormats/Serialization. I found a supposed fix for boost 1.69.00 that should work with the latest boost
https://github.com/daldegam/eos-portable-archive/pull/2
Using the latest boost the signals interface changes which causes other problems.


```
Package icu conflicts for:
root -> libxml2[version='>=2.9.8,<2.10.0a0'] -> icu[version='56.*|58.*|>=58.2,<59.0a0|>=64.2,<65.0a0']
root-dependencies -> libxml2[version='>=2.9.10,<2.10.0a0'] -> icu[version='>=64.2,<65.0a0']
boost=1.68.0 -> boost-cpp=1.68.0 -> icu[version='>=58.2,<59.0a0']
```

**Comment by gartung** [comment]
@Dr15Jones

**Comment by gartung** [comment]
Removing the linking of libboost_signals was the final hurdle to building with the latest boost. This resolved the conflicts from requiring boost=1.68.00.

**Comment by chrisburr** [comment]
You should only depend on root_base and root-binaries (I might have the hyphenation wrong). If you change that and retendre I can look at the error message and figure out what’s wrong

**Comment by gartung** [comment]
@chrisburr How do I rerender the .ci_support files and force the macos versions to 10.14?

**Comment by gartung** [comment]
@conda-forge-admin, please rerender

**Comment by gartung** [comment]
I think I figured out how to force 10.14. I needed to add conda_build_config.yaml

File: recipe/meta.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ requirements:
     - {{ cdt('libxdamage') }}  # [linux]
     - {{ cdt('libxxf86vm') }}  # [linux]
   host:
-    - boost =1.68.0
+    - boost
```
Comment (chrisburr) [inline recipe/meta.yaml]: Do you need `Boost.Python`? If not we should use:
```suggestion
    - boost-cpp
```

If so, you should:
- change the `- {{ pin_compatible('boost', max_pin='x.x') }}` line to match
- rerender the feedstock

**Comment by gartung** [comment]
@conda-forge-admin, please rerender

**Pull request merged**